storybook/stories/index.js

```diff
@@ -101,8 +97,12 @@ storiesOf('Components/Different', module)
   })
   .addDecorator((getStory) => <CenterView>{getStory()}</CenterView>)
   .add('Separator', () => <Separator />)
-  .add('SectionHeader title 1', () => <SectionHeader title='First title' />)
-  .add('SectionHeader title 2', () => <SectionHeader title='SECOND title' />)
+  .add('SectionHeaders', () => (
+    <React.Fragment>
```

Comment:
This indes.js should not contain additional JSX. Please move JSX into appropriate story file.